thread_options.h

```diff
@@ -699,9 +697,13 @@ struct thread_options_pack {
 
 	uint32_t log_entries;
 	uint32_t log_prio;
+
+	uint8_t patterns[];
```

Comment:
Should this be called buffer_patterns[] to be consistent with what came before?

Reply:
Well it's the amalgam of two variables; buffer_pattern and verify_pattern. 

So I would say buffer_patterns would be confusing as that's only one of the variables it was replacing.

Reply:
OK.